package.json

```diff
@@ -14,6 +14,7 @@
     "node": ">=16.15.1"
   },
   "type": "module",
+  "module": "./dist/launchdarkly-vue-client-sdk.es.js",
```

Comment:
Added based on the Vite docs.

Reply:
The recommended package.json part: https://v2.vitejs.dev/guide/build.html#library-mode